Patches/PowerArmour T-45b/T45b_CE_Patch.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>T-45b Power Armor</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<!-- ========== T-45b helmet ========== -->
```

Comment:
Should probably have `<SmokeSensitivity>-1</SmokeSensitivity>` added for the helmet.

Reply:
Fixed in 07ce4c7

Note that I have deliberately omitted the `WearingGasMask` ApparelHediffExtension, as I assume the T-45b Helmet from _Fallout 4_ is more comfortable than real-life gas masks.

Reply:
Perfectly reasonable.